my_game.py

```diff
@@ -128,14 +129,36 @@ def thrust(self):
             self.change_x *= player_x_and_y_speed_ratio
             self.change_y *= player_x_and_y_speed_ratio
 
-    def update(self):
+    def reset(self):
+        self.reset_timer = PLAYER_INVINCIBILTY_SECONDS
+        # The Player is Invisible
+        self.alpha = 0
+
+    @property
+    def invicible(self):
+        return self.reset_timer < 1
```

Comment:
That's confusing? The user sets _PLAYER_INVINCIBILTY_SECONDS_ to 3, but you are only invincible for 1 second?

Reply:
You need to add some comment somewhere about how this _invincibility/reset_ thing works.

Reply:
The code works as when you get hit by the asteroid you will disappear for 2 seconds.
After that you are invincible for 3 seconds, and you can get hit again.
You can see it in Player on_update.